packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -33,6 +33,13 @@ import {
 const { details: X } = assert;
 
 /**
+ * @typedef {{
+ * governedValues: {}, // ??? what type? ??? aren't there already subscriptions?
```

Comment:
Since governance is already on-chain, do we need to republish all the values or can we point to them somehow?

Also, @Chris-Hibbert WDYT of ParamManager having methods to support this?

Reply:
It absolutely makes sense for ParamManager to provide support. I'm waiting on more detailed requirements. 

I'm hoping to hear
 * what publishing/sharing mechanisms are needed
 * whether we want to emit events or statistics. At ${JOB-1} the s/w mostly said "x happened" and the monitoring systems did all the counting, averaging, etc. I can imagine we want more to happen in the contracts, so we'd emit counts, averages, maximums, etc.